### ISSUE TITLE
No temporal evolution when updating entity attributes

### ISSUE THREAD

**Issue by Blobonat** [ScorpioBroker/ScorpioBroker#339]
Hi,

when updating an attribute via the `entities/<ID>/attrs/<attId>` endpoint including an `obversedAt` value the temporal represenation of the entity is not updated.

Tested version: scorpiobroker/all-in-one-runner:spring-kafka-latest

Steps to reproduce:
- Create entity with temporal property
```
curl --location --request POST 'http://localhost:9090/ngsi-ld/v1/entities' \
--header 'Content-Type: application/json' \
--header 'NGSILD-Tenant: openiot' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
    "id": "urn:ngsi-ld:Vehicle:A123",
    "type": "Vehicle",
    "brandName": {
        "type": "Property",
        "value": "Mercedes"
    },
    "speed": {
        "type": "Property",
        "value": 80,
        "observedAt": "2022-09-10T12:21:24.000Z"
    }
}'
```
- Update temporal property via `entities/<ID>/attrs/<attId>` endpoint
```
curl --location --request PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A123/attrs/speed' \
--header 'NGSILD-Tenant: openiot' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--header 'Content-Type: application/json' \
--data-raw '{
    "type": "Property",
    "value": 91,
    "observedAt": "2022-09-10T13:21:24.000Z"
}'
```
- Make temporal query
```
curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/temporal/entities/urn:ngsi-ld:Vehicle:A1000?attrs=speed&timerel=between&timeAt=2021-10-10T15:24:10.000Z&endTimeAt=2022-12-10T19:23:12.000Z' \
--header 'NGSILD-Tenant: openiot' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--header 'Content-Type: application/json'
```
- Expected behavior: temporal response includes two values for `speed`
- Actual behavior: temporal response only includes initial value for `speed`
```
{
    "id": "urn:ngsi-ld:Vehicle:A1000",
    "type": "Vehicle",
    "speed": {
        "type": "Property",
        "value": 80,
        "instanceId": "urn:ngsi-ld:bc181360-575c-4fed-9365-fb1456b3027d",
        "observedAt": "2022-09-10T12:21:24.000Z"
    }
}
```

Using the `entityOperations/upsert/` endpoint to update the `speed` property correctly updates the temporal representation.

**Comment by amitrnec** [comment]
Hi,
Thank you for the information.
I tried to reproduce this issue but it is working fine as expected result.
Tested version: development-quarkus
We will try to test with "scorpiobroker/all-in-one-runner:spring-kafka-latest" version also.

Thanks,
Amit.

**Comment by amitrnec** [comment]
Hi,
Now we have fixed this issue #342


Thanks

**Comment by Blobonat** [comment]
Thank you!
